### DISCUSSION TITLE
Add file extension for "Markdown" language?

### DISCUSSION THREAD

**Discussion by gcoyle83** [streetsidesoftware/vscode-spell-checker#2980] [Q&A] [answered]
This extension works well for Markdown files with ".md" extension. However, I'd like it to work on Quarto markdown files, as well, which have ".qmd" extension. Is it possible to associate ".qmd" with the Markdown language somewhere in the extension settings?

**Comment by Jason3S** [comment, marked as answer]
It is possible to add new file types. The name of the command is a bit vague.

## Using a command
Steps:
1. Open a Quarto markdown file
2. `F1` -> `Spell: Enable Spell Checking Document Language`

<img width="625" alt="image" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/eaf1db46-fc01-414d-9d27-048048e92b0f">

This will up VSCode settings.

## From VSCode settings

Steps:
1. Open VSCode Settings
2. Search for `spell Enable Filetypes`
3. Add `quarto` to the list.

<img width="1095" alt="image" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/17d214d5-c5ea-4a09-9c71-f217f754f6d5">

**Reply by gcoyle83** [reply]
Great! Thanks for such a quick reply, and a very helpful extension.
